src/test/java/net/minecraftforge/debug/HorseArmorTest.java

```diff
@@ -0,0 +1,73 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.entity.passive.EntityHorse;
+import net.minecraft.entity.passive.HorseArmorType;
+import net.minecraft.init.MobEffects;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.potion.PotionEffect;
+import net.minecraft.world.World;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.common.IHorseArmor;
+import net.minecraftforge.common.util.EnumHelper;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventBusSubscriber;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry.ObjectHolder;
+
+@Mod(modid = HorseArmorTest.MODID, name = "HorseArmorTest", version = "1.0", acceptableRemoteVersions = "*")
+public class HorseArmorTest 
+{
+    public static final String MODID = "horse_armor_test";
+    public static final boolean ENABLED = false;
+    
+    public static HorseArmorType testArmorType;
+    @ObjectHolder(MODID + ":test_armor")
```

Comment:
I mean with the holder. Although in reality it shouldn't be the case that test mod blocks/items get overridden, they should reflect best practices. The only minor nitpick I'd have is that the annotation is used on a field level rather than a class one, which is preferred since it requires fewer "magic strings".